content/source/docs/enterprise/api/index.html.md

```diff
@@ -161,6 +161,33 @@ $ curl \
 }
 ```
 
+## Rate Limiting
+
+You can make up to 30 requests per second to the API as an authenticated or unauthenticated request. If you reach the rate limit then you will be throttled and an error response will be returned.
+
+Authenticated requests are allocated to the user associated with the authentication token. This means that a user with multiple tokens will still be limited to 30 requests per second, additional tokens will not allow you to increase the requests per second permitted.
+
+Unauthenticated requests are associated with the requesting IP address.
```

Comment:
I had no idea we allowed unauthenticated requests! What endpoints can an anon hit?

Reply:
@nfagerlund if you just hit an endpoint unauthenticated it still goes through to the application so all this will do is throttle those requests based on IP. I don't think we specifically have endpoints that can be accessed unauthenticated.